verify_git_config.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
Can we put this into contrib/scripts or so to not litter the root folder?

Reply:
Yep, done